### PR TITLE
Doc / Check for Centos restart

### LINKED ISSUE #622: Directions for RHEL/CentOS
According to [this page](https://support.rackspace.com/how-to/centos-hostname-change/) the way to re-read the hosts file on the above OSes is

```
$ sudo /etc/init.d/network restart
```

Can we get this added to the readme?

### PR DESCRIPTION
**Pull request by gfyoung**
Closes #622.

### CODE REVIEW

**Comment by StevenBlack** [comment]
Thanks @gfyoung !

Merging.

File: updateHostsFile.py [hunk 1]
```diff
@@ -1063,6 +1063,15 @@ def flush_dns_cache():
                 else:
                     print_success(nscd_msg.format(result="succeeded"))
 
+        centos_file = "/etc/init.d/network"
```
Comment (funilrys) [inline updateHostsFile.py]: May I ask if we are sure that this file is only present into CentOS ?

Comment (gfyoung) [reply updateHostsFile.py]: I'm not aware of any other distros at the moment.

Comment (funilrys) [reply updateHostsFile.py]: `Redhat`, `Fedora`, `Suse` and `OpenSuse Linux` ?

 I'm pretty sure there was (and maybe still is) a `/etc/init.d/network` into those distros.

Comment (gfyoung) [reply updateHostsFile.py]: Ah, that's right.  In the docs, it's referred to as `RHEL / Centos` (though obviously such a variable name isn't possible in Python).

**Pull request merged**